src/cockpit/bridge.py

```diff
@@ -216,7 +216,9 @@ def main() -> None:
         print(json.dumps(Packages().get_bridge_configs(), indent=2))
     else:
         asyncio.set_event_loop_policy(EventLoopPolicy())
-        asyncio.run(run(args), debug=args.debug)
+        asyncio.get_event_loop().set_debug(args.debug)
```

Comment:
This is pretty evil.  It might not be clear, but this actually creates the loop.  This usage is deprecated.  See https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop

We pass the `args` into the run() function anyway so it's probably safer to call `get_running_loop()` from there and `.set_debug()` on that.

Reply:
Thanks for spotting! Reworked, how's this?

Reply:
interdiff, before the rebase to mypy PR: https://github.com/cockpit-project/cockpit/compare/ef474e1ab55b133724461f3689fac492416bcd8c..6bd43c82e12d4354990ab1e0454586e52dcdf37a